hydromt_wflow/wflow_sediment.py

```diff
@@ -244,88 +250,63 @@ def setup_outlets(
 
     def setup_gauges(
         self,
-        gauges_fn=None,
-        source_gdf=None,
-        snap_to_river=True,
-        mask=None,
-        derive_subcatch=False,
-        basename=None,
-        toml_output="csv",
-        gauge_toml_header=["Q", "TSS"],
-        gauge_toml_param=["lateral.river.q_riv", "lateral.river.SSconc"],
+        gauges_fn: Union[str, Path, gpd.GeoDataFrame],
+        index_col: Optional[str] = None,
+        snap_to_river: Optional[bool] = True,
+        mask: Optional[np.ndarray] = None,
+        snap_uparea: Optional[bool] = False,
+        max_dist: Optional[float] = 10e3,
+        wdw: Optional[int] = 3,
+        rel_error: Optional[float] = 0.05,
+        derive_subcatch: Optional[bool] = False,
```

Comment:
```suggestion
        abs_error: Optional[float] = 50.0,
        fillna: Optional[bool] = False,
        derive_subcatch: Optional[bool] = False,
```

New arguments from `setup_gauges` in `WflowModel`

Reply:
Good catch!